addon/components/cfb-code-editor.hbs

```diff
@@ -0,0 +1,13 @@
+<div>
```

Comment:
I had to co-locate the template because with glimmer components templates in addons are not found during testing (see https://discuss.emberjs.com/t/how-to-use-glimmer-components-in-addon/17464). Should I use a classic component instead?

Reply:
fixed, I forgot the template export in `app/`